src/Data/Pool.hs

```diff
@@ -102,7 +102,7 @@ createPool create free numStripes idleTime maxResources = newPool PoolConfig
   { createResource   = create
   , freeResource     = free
   , poolCacheTTL     = realToFrac idleTime
-  , poolMaxResources = numStripes * maxResources
+  , poolMaxResources = maxResources
```

Comment:
This change breaks compatibility. The variable name `maxResources` means max resources *per stripe* in resource-pool(<0.3)

https://github.com/scrive/pool/blob/f4be4fbe253ac2927d62153a79df9b5957c125e2/Data/Pool.hs#L147

```suggestion
  , poolMaxResources = numStripes * maxResources
```